### DISCUSSION TITLE
The ability to go to the related record using recordfinder

### DISCUSSION THREAD

**Discussion by AIC-BV** [wintercms/winter#482] [Ideas]
It would be nice that when using 'Record finder' (https://wintercms.com/docs/backend/forms#widget-recordfinder), you can actually open the related record. Now you have to manually go and find the related record if you wish to see what it's actually related to.

Currently, I added a partial to link to the related record (example for user relation).
```
<a href='<?= e(\Backend::url("winter/user/users/update/{$model->user->id}")); ?>' target='_blank'>
    Link to user
</a>
```
Would be nice if something like this was built in!

For example:

1. Contact model has a user_id.
2. User 1 filled in the contact form.
3. The backend administrator notices a new Contact ticket
4. The Contact ticket has user_id 1. This is linked with a recordfinder (ID 1).
**NOW**: The administrator can't fnd the e-mailadress to contact the user at the Contact ticket. The administator must search the Users list to find the e-mailadress (or any other extended user field that isn't in the User columns.yaml).
**NICE TO HAVE**: The administrator can click on the recordfinder to go straight to the user-details (I prefer target _blank)

**Comment by LukeTowers** [comment]
I like this idea, `recordUrl` and `recordOnClick` with support for arbitrary related record attributes in the colon syntax (i.e. `:id`) would be an accepted PR if anyone wants to work on this. See the Lists widget for how those values are currently implemented.
